core/src/main/cpp/msg/MessageHeader.h

```diff
@@ -1,3 +1,19 @@
+/**
+ * Copyright (c) 2016-2018 Netflix, Inc.  All rights reserved.
```

Comment:
If the copyright text was missing, we should make sure the start year is correct.

Reply:
@padolph was it 2016 when you first created this file?

Reply:
Checked file history. 2016 is the correct start year.